src/lesson13/Main.java

```diff
@@ -0,0 +1,66 @@
+package lesson13;
+
+import java.util.*;
+
+public class Main {
+    public static void main(String[] args) {
+
+//1
+        String string = "What is Java technology and why do I need it? Java is a programming language and computing platform first released by Sun Microsystems in 1995. It has evolved from humble beginnings to power a large share of today’s digital world, by providing the reliable platform upon which many services and applications are built. New, innovative products and digital services designed for the future continue to rely on Java, as well.";
+        Collection<String> words = new ArrayList<>(List.of(string.split(" ")));
+        System.out.println(words.size()); // number of words = 70;
```

Comment:
так буде правильніше. Змінну створити типу List, а в нього заюзатии лістІтератор
```suggestion
        List<String> words = new ArrayList<>(List.of(string.split(" ")));
        System.out.println(words.size()); // number of words = 70;
```